strapi/strapi
# enhancement(content-releases): add new status column to Release Details

⚠️ ⚠️  **Important:** This PR doesn't handle validation messages yet, I have some problems related to what validator we should use on Content-Releases, so, we decided to go with only the status (if entry is publish/unpublish) without validations for the beta and try to find the best solution to add validations for the stable release of content-releases.

### What does it do?

Add a new "status" column to give user info about the entry status (If it's ready to publish/unpublish or it's already publish/unpublished)

### How to test it?

You should have Content-Releases enabled and a right EE license.
1. Go to any release with entries attached
2. You should see a new "Status" column.
- If entry is in `draft` and the action is `publish`, it should say: `Ready to publish`
- If entry is `published` and the action is `publish`, it should say: `Already published`
- If entry is in `draft` and the action is `unpublish`, it should say: `Ready to unpublish`,
- If entry is `published` and the action is `unpublish`, it should say: `Ready to unpublish`
